draft-ietf-teep-protocol.md

```diff
@@ -290,7 +290,7 @@ the listed steps fail, then the TEEP message MUST be rejected.
   supported or that are specified as being ignored when not
   understood.
 
-1. Follow the steps specified in Section 4 of {{RFC8152}} ("Signing Objects") for
+1. Follow the steps specified in {{Section 4 of RFC9052}} ("Signing Objects") for
```

Comment:
I don't think this syntax works does it?  The thing in `{{}}` has to be an anchor

Reply:
If you are using a reasonably recent version of kramdown-rfc, it should work: see

* [kramdown-rfc2629.rb#L138](https://github.com/cabo/kramdown-rfc/blob/93427e543b09bbdd651cdac90aa930eb609fe2ac/lib/kramdown-rfc2629.rb#L138), and
* [kramdown-rfc2629.rb#L66](https://github.com/cabo/kramdown-rfc/blob/93427e543b09bbdd651cdac90aa930eb609fe2ac/lib/kramdown-rfc2629.rb#L66)